app/views/works/index.html.erb

```diff
@@ -0,0 +1,78 @@
+<h2>List of Works</h2>
+
+<section> 
+  <article class="media-table">
+    <h4>Albums</h4>
+    <table class="table">
+      <tr>
+        <th width="80">Votes</th>
+        <th>Title</th>
+        <th>Created By</th>
+        <th width="100">Published</th>
+        <th width="100">Upvote</th>
+      </tr>  
+      <% Work.index_order("album").each do |work|  %>
+        <tr>
+          <th><%= work[1] %></th>
+          <th><%= link_to Work.top_ten_helper(work[0]).title, work_path(Work.top_ten_helper(work[0])) %></th>
+          <th><%= Work.top_ten_helper(work[0]).creator %></th>
+          <th><%= Work.top_ten_helper(work[0]).publication_year %></th>
+          <th><%= button_to "Upvote", upvote_work_path(Work.top_ten_helper(work[0])), class: "btn btn-primary", rel: "nofollow" %></th>
+        </tr>  
+      <% end %>
```

Comment:
Hey do you notice you're doing this list a lot, you can create a partial view of each of these lists and just pass the partial the collection of works you want it to render.